app/cinema/bar.py

```diff
@@ -0,0 +1,4 @@
+class CinemaBar:
+    @staticmethod
+    def sell_product(product: str, customer: str) -> None:

```

Comment:
Invalid type hint for `customer`.
```suggestion
    def sell_product(product: str, customer: Customer) -> None:

```